.github/workflows/component-bumper.yml

```diff
@@ -13,6 +13,11 @@ jobs:
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+      with:
+        ref: master
```

Comment:
What happens once we branch out? Do I need to change this on the branch out? Can we take the branch as a variable from actions?

Reply:
do you think we would ever run bumper script from a branch that is not master?
The way I see it, the only case it might be need to do so is on the stable branch, and in that case I thought we would actually prefer to /backport it. what do you think?

Reply:
Bumper script on release branch is super difficult since we really don't know what components branch do we have to look at, maybe we can put that information at components.yaml.

Reply:
components.yaml is pointing to the branch we take patches from.

On branch-out of CNAO, we have couple of options:
- Branch out a component for stable release and pin components.yaml to that branch
- Keep taking component from master and only once we are fine with the state of the stable branch, turn it "static" in components.yaml

If you expect this to be too difficult, we can limit this script to the master for now.

Reply:
if we branch out, then the components.yaml can be different, but if we define that bumper only runs on master and only backport the component-bumps PR we consider safe for stable branch, then I'm not sure it's really needed.
Or maybe I don't catch your drift. Let's further discuss

Reply:
While master would follow minor releases of our sub-components, stable branch would follow patch releases. So there may be nothing to backport.

Anyway, only now I realized this PR is indeed only working with master. We can follow-up about stable branches later.

Reply:
ack. I will work on it on a separate PR, after I make sure this rebase works well